drivers/espi/Kconfig.it8xxx2

```
@@ -21,6 +21,81 @@ config ESPI_PERIPHERAL_HOST_IO
 config ESPI_PERIPHERAL_DEBUG_PORT_80
 	default y
 
+config ESPI_PERIPHERAL_EC_HOST_CMD
+	default y
+
+choice IT8XXX2_H2RAM_HC_SIZE_CHOICE
+	prompt "choose a H2RAM space for ec host command"
+	depends on ESPI_PERIPHERAL_EC_HOST_CMD
+config IT8XXX2_H2RAM_HC_SIZE_256
```

You don't need to place `config IT8XXX2_H2RAM_HC_SIZE_256`, you can order the options ascending, and the specify the default as:
```
choice IT8XXX2_H2RAM_HC_SIZE_CHOICE
	prompt "H2RAM space for ec host command"
	default IT8XXX2_H2RAM_HC_SIZE_256
	depends on ESPI_PERIPHERAL_EC_HOST_CMD
```

Also add an empty line for readability.
```suggestion
 
config IT8XXX2_H2RAM_HC_SIZE_256
```

---

Done.